clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -759,9 +855,27 @@ void darwin::Linker::ConstructJob(Compilation &C, const JobAction &JA,
                        "-filelist"};
   }
 
+  // Set the environment to include ZERO_AR_FLAGS.
+  //
+  // FIXME: Add a flag to the linkers to support this without an environment
```

Comment:
Another reproducibility related upstreaming. Do we need this?

Reply:
Seems useful.